quarkusQS/src/main/java/my/groupId/DemoRestApplication.java

```diff
@@ -0,0 +1,11 @@
+package my.groupId;
```

Comment:
Can we have a diffetent package naming? What is common in quarkus quickstarts/examples?

Reply:
I have seen `org.acme` for most projects in [QuarkusQS repo](https://github.com/quarkusio/quarkus-quickstarts)